README.md

```diff
@@ -169,8 +169,8 @@ Using `--help` will display the `pathfinder` options, including their environmen
 # with built from source
 cargo run --release --bin pathfinder -- --help
 
-# with docker images
-docker run --rm eqlabs/pathfinder /usr/local/bin/pathfinder --help
+# with docker images (0.1.12 onwards)
```

Comment:
If it's a breaking change we should probably move to `0.2.1`?

Reply:
We haven't yet released 0.2.0 but otherwise would be a good idea.